src/include/distributed/insert_select_executor.h

```diff
@@ -0,0 +1,19 @@
+/*-------------------------------------------------------------------------
+ *
+ * insert_select_executor.h
+ *
+ * Declarations for public functions and types related to executing
+ * INSERT..SELECT commands.
+ *
+ * Copyright (c) 2017, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#pragma once
```

Comment:
It seems we don't use `pragma once` any other place? Instead, we use `ifndef ..` (Same comment for the planner header file as well.)